docs/docs/gatsby-agency-partnership.md

```diff
@@ -2,21 +2,15 @@
 title: Agency Partnership Program
```

Comment:
This title is different is now "Agency Partner Program in the navigation. Should this be changed as well?

Reply:
Let's make them both 'Agency Partner Program'. thanks for checking that.